tests/openedx_tagging/core/tagging/test_models.py

```diff
@@ -29,7 +29,7 @@ def setUp(self):
         self.taxonomy = Taxonomy.objects.get(name="Life on Earth")
         self.system_taxonomy = Taxonomy.objects.get(
             name="System defined taxonomy"
-        ).cast()
+        )
```

Comment:
Suggested removing this so that the test_rules are testing with an un-cast system taxonomy, so we know the rules are handling the cast.